src/nagendra/Mall.java

```diff
@@ -0,0 +1,60 @@
+package nagendra;
+
+public class Mall {
+
+	int numberOfItems;
+	String customername;
+	String paymentType;
+	String section;
+	boolean isBillRequired;
+	boolean doesCustomerNeedCarryBag;
+
+	void itemAddedToCart(int numberOfItems) {
+		this.numberOfItems = numberOfItems;
+	}
+
+	void customerName(String customername) {
+		this.customername = customername;
+	}
+
+	void paymentDoneUsing(String paymentType) {
+		this.paymentType = paymentType;
+	}
+
+	void displayCutomerInfo() {
+		System.out.println(customername + " brought " + numberOfItems + " items and payment done using " + paymentType
+				+ " in " + section);
+	}
+
+	void needBill(boolean isBillRequired) {
+		this.isBillRequired = isBillRequired;
+	}
+
+	void haveCarryBag(boolean doesCustomerNeedCarryBag) {
+		this.doesCustomerNeedCarryBag = doesCustomerNeedCarryBag;
+	}
+
+	void decideSection() {
+		if (numberOfItems < 10 && paymentType.equals("cash")) {
+			section = "section 1A";
+		} else if (numberOfItems > 10 && paymentType.equals("visa card")) {
+			section = "section 1B";
+		} else if (numberOfItems > 10 && paymentType.equals("cash")) {
+			section = "section 1c";
+
+		} else {
```

Comment:
You should add condition for visa card(4th condition), "else" will execute always if customer passing other value than visa & cash, we need to add that 4th condition.
This is also fine for now, my point is to you know other scenarios